README.md

```diff
@@ -11,6 +11,19 @@ It is published online at: [docs.ubuntu.com/en/styleguide](https://docs.ubuntu.c
 The Vale linter operates from a series of rules. These are defined in individual YAML files, grouped into 'Styles'.
 This repository contains the Canonical set of rules, or the Canonical Style. 
 
+### Manual check
+
+To manually check your documentation with Vale rules use the following steps:
+
+1. Install Vale.
+2. Clone this repository.
+3. Run Vale with the config from this repository for testing your documentation source files: 
+```shell
+vale --config ~/test/praecepta/vale.ini ~/test/karapace-operator/docs/
+```
```

Comment:
This probably needs indentation (since it's part of step 3).

Reply:
Fixed in https://github.com/canonical/praecepta/pull/68/commits/1b3994b1e08d9aedb43a5880e7c68d8c7363b200